modules/op-defining-and-creating-pipelineresources.adoc

```diff
@@ -0,0 +1,76 @@
+// Ths module is included in the following assembly:
+//
+// *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
+
+[id="defining-and-creating-pipelineresources_{context}"]
+= Defining and creating PipelineResources
+
+_PipelineResources_ are artifacts that are used as inputs or outputs of a Task.
+
+After you create Tasks, create PipelineResources that contain the specifics of the Git repository and the image registry to be used in the Pipeline during execution:
+[NOTE]
+====
+If you are not in the `pipelines-tutorial` namespace, and are using another namespace, ensure you update the front-end and back-end image resource to the correct URL with your namespace in the steps below. For example:
+----
+image-registry.openshift-image-registry.svc:5000/<namespace-name>/vote-api:latest
+----
+====
+[discrete]
+.Procedure
+. Create a PipelineResource that defines the Git repository for the front-end application:
++
+----
+
+ $ tkn resource create
+ ? Enter a name for a pipeline resource : ui-repo
+ ? Select a resource type to create : git
+ ? Enter a value for url : http://github.com/openshift-pipelines/vote-ui.git
+ ? Enter a value for revision : release-tech-preview-1
+ New git resource "ui-repo" has been created
```

Comment:
Per guidelines, please separate output into separate copy block here and for back end (44).